api-reference/beta/api/device_delete.md

```diff
@@ -2,14 +2,15 @@
 
 > **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
 
-Delete device.
+Delete a registered device.
+
 ## Permissions
 One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
 
 
 |Permission type      | Permissions (from least to most privileged)              |
 |:--------------------|:---------------------------------------------------------|
-|Delegated (work or school account) | Directory.AccessAsUser.All    |
+|Delegated (work or school account) | Directory.ReadWrite.All, Directory.AccessAsUser.All |
```

Comment:
Are you sure that Directory.ReadWrite.All allows delete?  I'm surprised by this.

Reply:
Good catch.  I copied it over from v1.0.  It doesn't work.  I removed it from both.

Reply:
Forgot to mention you...
@dkershaw10 Good catch. I copied it over from v1.0. It doesn't work. I removed it from both.